extensions/extensions-helm/aissemble-kafka-chart/values.yaml

```diff
@@ -2,7 +2,6 @@
 ## CONFIG | Kafka Configs
 ########################################
 kafka:
-  kafkaVersion: 3.8.0
```

Comment:
Q: no more version required?

Reply:
This parameter is not valid in the base chart at all and we have no templates. So it's a completely unused property.